kobo-uncaged/main.go

```diff
@@ -719,6 +720,7 @@ func (ku *KoboUncaged) SaveBook(md map[string]interface{}, len int, lastBook boo
 	// The calibre wireless driver does not sanitize the filepath for us. We sanitize it here,
 	// and if lpath changes, inform Calibre of the new lpath.
 	newLpath = ku.invalidCharsRegex.ReplaceAllString(koboMD.Lpath, "_")
+	newLpath = lpathKepubConvert(newLpath)
```

Comment:
Same here.

Reply:
I kind of want to keep this one as-is, because it's doing two different checks/conversions for different reasons.

I might comment the second call though.